### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by anthony-crisp**
Avatar now shows next to posts and comments.

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -22,7 +22,7 @@
     <div>
       <h3>Comments</h3>
       <% post.comments.each do |comment| %>
-        <p> <%= comment.user.name %> said: <%= comment.body %></p>
+        <p> <%= image_tag post.user.avatar.to_s, size: User::AVATAR_SIZES[:small] %> <%= comment.user.name %> said: <%= comment.body %></p>
```
Comment (adamjohnsnow) [inline app/views/posts/index.html.erb]: Is this right - should it not be the comment.user.avatar here?

**Review by adamjohnsnow — CHANGES_REQUESTED** [review]
See comment to line 25

**Review by adamjohnsnow — APPROVED** [review]
Looks good. Over to @neoeno

**Comment by neoeno** [comment]
Update this to the latest master and you'll have a test failure — a canary in the coal mine for breaking master due to old DB stuff! Have a go at fixing that.

I think it might be nice to somehow modify `User` so that no one can make this mistake again!

**Comment by neoeno** [comment]
Actually @JayWebDevCom and @sulaimancode just told me they're going to fix the whole issue! So you only need to wait for them to do that.

**Comment by neoeno** [comment]
@adamjohnsnow or @JayWebDevCom can you confirm this isn't going to cause any issues in production now? Then we can go ahead and merge!

**Review by neoeno — APPROVED** [review]

**Comment by adamjohnsnow** [comment]
@neoeno The migrations sorted orphaned posts and comments but I don't think this will now work as some users won't have avatars...

**Comment by neoeno** [comment]
We have default avatars though right?

**Comment by adamjohnsnow** [comment]
No. I'll get on that now.

**Comment by neoeno** [comment]
```
2.4.0 :005 > Post.all.first.user.avatar.to_s
  Post Load (0.5ms)  SELECT  "posts".* FROM "posts" ORDER BY "posts"."id" ASC LIMIT $1  [["LIMIT", 1]]
  User Load (0.3ms)  SELECT  "users".* FROM "users" WHERE "users"."id" = $1 LIMIT $2  [["id", 2], ["LIMIT", 1]]
 => ""
```

No errors on the web interface either! I say merge.

**Pull request merged**